src/CheckSessionIFrame.ts

```diff
@@ -55,6 +57,10 @@ export class CheckSessionIFrame {
                 if (this._stopOnError) {
                     this.stop();
                 }
+
+                if (this.propagateUserSessionError) {
```

Comment:
I guess we could combine `propagateUserSessionError` and `userSessionErrorCallback`: Make `userSessionErrorCallback` optional, if defined call it if not not. Then `propagateUserSessionError`  is not needed inside this class...

Reply:
Actually the way I implemented it is the same pattern as the `_callback` is called after the `changed` status is triggered. By using the `propagateUserSessionError ` flag it is made optional. If you'd rather like that the existence of the `userSessionErrorCallback` is checked to trigger that callback I would have to allow for a function to be defined in the settings object which, of course, would be possible but would break with the way that a client can attach to e.g. the `changed` status.